api/handlers.go

```diff
@@ -32,7 +32,7 @@ import (
 const InvalidParamsMessage = "Invalid query or path parameters"
 
 // jetIDRegexp uses for a validation of the JetID
-var jetIDRegexp = regexp.MustCompile(`^[0-1]{1,216}$`)
+var jetIDRegexp = regexp.MustCompile(`^(\*|([0-1]{1,216}))$`)
 
```

Comment:
Should we change jetID value at `JetDropsByJetID` too?

Reply:
And what about this?

Reply:
jetIDRegexp need to check incoming value from frontend. 
incoming value can be single "*" or numbers where length > 0. 
so, we need to left `^(\*|([0-1]{1,216}))$` expression.

Reply:
Yes, I'm talking about  `JetDropsByJetID` func in this file (I can't comment on this function, cause it wasn't changed, so left comment here, at same file). I think we need to replace jetID param at `JetDropsByJetID` func from '*' to empty string.